### PR TITLE
Ensure data value x is always a number

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Fixes chart issues with wrong data values.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/charts/src/ChartFactory.js [hunk 1]
```diff
@@ -48,12 +48,15 @@ const ChartFactory = React.memo(
...
     },
     disableShowMore,
-    data,
+    data: propData,
     isComparison,
     comparisonData,
     referenceData,
     profiles
   }) => {
+    const data = useMemo(() => propData.map(d => ({ ...d, x: Number(d.x) })), [
```
Comment (kilemensi) [inline packages/charts/src/ChartFactory.js:57]: `text.split` in `computeMaxLabelDimension` expects this to be a `string` (default `Card` and `ChartFactory` stories fails).

Comment (karimkawambwa) [reply packages/charts/src/ChartFactory.js:57]: Yeah, looking through it now

File: packages/charts/src/WrapLabel/wrapSVGText.js [hunk 2]
```diff
@@ -10,10 +10,11 @@ export function computeMaxLabelDimension({ labelWidth, texts }) {
   document.body.appendChild(svg);
   svg.appendChild(textEl);
 
-  const text =
+  const text = String(
```
Comment (kilemensi) [inline packages/charts/src/WrapLabel/wrapSVGText.js:13]: Works but why not ES6 templates or even just plain [concatenation](https://stackoverflow.com/questions/29055518/are-es6-template-literals-faster-than-string-concatenation)?

Comment (karimkawambwa) [reply packages/charts/src/WrapLabel/wrapSVGText.js:13]: Will keep this in mind.

**Pull request merged**